### ISSUE TITLE
How to use my own dataset for training?

### ISSUE THREAD

**Issue by xiezufeng** [seasonSH/SemanticStyleGAN#16]
How can I use my own dataset for training? Or can I use my own data set for training?

**Comment by seasonSH** [comment]
Yes you can. You can create your own dataset by following the data preparation section in README

**Comment by xiezufeng** [comment]
OK, thank you for your patience
